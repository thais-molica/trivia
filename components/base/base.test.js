import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Base from "./index";

it("Base renders correctly", () => {
  const tree = renderer.create(<Base><div>teste</div></Base>).toJSON();
  expect(tree).toMatchSnapshot();
});
