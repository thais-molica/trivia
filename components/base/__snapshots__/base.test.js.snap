// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base renders correctly 1`] = `
Array [
  <header
    className="jsx-9438985"
  >
    <a
      className="jsx-9438985"
      href="/"
    >
      Teste Dev Frontend
    </a>
  </header>,
  <div
    className="jsx-3040147036 "
  >
    <div>
      teste
    </div>
  </div>,
]
`;
